scripts/uart_terminal/uart_terminal.py

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+import sys
+import subprocess
+import os
+
+absolutePath = os.path.join(os.path.abspath(os.getcwd()), "scripts")
+sys.path.append(absolutePath)
+from open_putty import open_putty
+
+print("Configure and open terminal(s)\n")
+
+if sys.platform == "linux":
+    terminator = subprocess.Popen(
+        ["terminator", "--config=scripts/linux_terminator_config"], stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+elif sys.platform == "win32":
+    open_putty()
+else:
+    print("OS not supported")
```

Comment:
we should have a solution that also work for macOS as this is one of our supported OS'es.
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_recommended_versions.html#supported-operating-systems

Reply:
Agreed, will implement for macOS.